conduit/plugins/metadata.go

```diff
@@ -1,5 +1,20 @@
 package plugins
 
+// Metadata returns fields relevant to identification and description of plugins.
+type Metadata struct {
+	Name         string
+	Description  string
+	Deprecated   bool
+	SampleConfig string
+	Type         PluginType
```

Comment:
I don't see where `Type` is used. Looks like it was previously unused as part of `PluginMetadata` and now it's still unused as part of `Metadata`?